outputs/client.go

```diff
@@ -178,6 +178,12 @@ func (c *Client) Post(payload interface{}) error {
 		req.Header.Add("event-namespace", c.Config.Kubeless.Namespace)
 	}
 
+	if c.OutputType == "GCPCloudRun" {
+		if c.Config.GCP.CloudRun.JWT != "" {
```

Comment:
I made that control for users who might want to use the workload identity feature instead of JWT based authentication. So, JWT based authentication should not be required for us, but if the users want JWT based authentication to enable, we should add it to the Authorization header.